### ISSUE TITLE
Default or expose a method to use latest versions of rules

### ISSUE THREAD

**Issue by adammscisz** [phax/ph-bdve-rules#2]
Currently when new rules appear, one needs to change both: update *-rules dependency and and pass specific VESID ruleset to Registry.

It would be nice to have an option to use latest ruleset by default so that pom update is enough to be up to date with changes.

**Comment by phax** [comment]
That would work for all validation artefacts but may require a date for selection, in case a validation artefact is published and included before it is in effect.

**Comment by fabmoll** [comment]
I was just wondering how to always test the latest rules.  I hope you'll be able to add it :)

**Comment by fabmoll** [comment]
Hello phax.  Do you a date in mind to add this feature ? :)  Thank you for your work!

**Comment by phax** [comment]
Yes, I have that in the back of my head. Like a lot of other things... sorry 🤷🏼

**Comment by fabmoll** [comment]
Hello Philip, 

Do you think you'll add that feature in coming month ? it could be really useful for us :)
I know it's not a top priority and I'm sorry to bother you with that.

Thank you Philip for the work you do in the API.
Have a nice day.

Fabrice.

**Comment by phax** [comment]
@fabmoll yes, noted :)
